src/app/loginComponents/accounts/internal/change-username/change-username.component.scss

```diff
@@ -1,20 +1,61 @@
+@use '@angular/material' as mat;
+
+@import '/src/materialColorScheme.scss';
+
 .change-username-form {
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 }
 
-.change-username-form-buttons {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+mat-form-field {
+  width: 60%;
 }
 
-#updateUsername {
-  width: 200px;
+mat-error {
+  line-height: 2rem;
 }
 
 .vertical-center {
   display: flex;
   align-items: center;
 }
+
+// Form field inner
+:host ::ng-deep .mat-form-field-wrapper .mat-form-field-infix {
+  border: 0rem;
+  padding: 0rem;
+}
+
+//Form field box
+:host ::ng-deep .mat-form-field-wrapper {
+  margin: 0rem;
+  padding-bottom: 0.8rem;
+  line-height: 4rem;
+}
+
+// Form field label
+:host ::ng-deep .mat-form-field-wrapper .mat-form-field-flex .mat-form-field-infix .mat-form-field-label-wrapper .mat-form-field-label {
+  top: 1.8rem;
+}
+
+// Form field text animation re-implementation
```

Comment:
Why did this need to be re-implemented? I'm not sure, so maybe an additional comment would be helpful

Reply:
The placeholder text "Username" moves from the form field to the top border - when you change the height of the form field box the animation gets messed up, the only solution I've seen is to re-implement the animation. I needed to do the same with the search page form field

![image](https://user-images.githubusercontent.com/97123241/165621558-a897c306-da52-4424-9888-a943ffd4115d.png)
![image](https://user-images.githubusercontent.com/97123241/165621600-581e77e6-91fb-4436-b593-85d9a8b3366b.png)

I'll add a second comment on this, thanks kathy